packages/docs-ui/src/services/doc-popup-manager.service.ts

```diff
@@ -57,11 +57,67 @@ export function transformPosition2Offset(x: number, y: number, scene: Scene) {
     };
 }
 
+// write a revert function for transformPosition2Offset
```

Comment:
Is it a TODO or something? 
```suggestion
// TODO@weird94: write a revert function for transformPosition2Offset
```